./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
./auto_run.sh: line 11: shuf: command not found
./auto_run.sh: line 12: shuf: command not found
./auto_run.sh: line 13: shuf: command not found
./auto_run.sh: line 14: shuf: command not found
./auto_run.sh: line 15: shuf: command not found
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "../../revsim/auto_ga.py", line 4, in ?
    from revsim import *
  File "/home/lowr4210/GA/revsim/revsim.py", line 13, in ?
    from real_io import *
  File "/home/lowr4210/GA/revsim/real_io.py", line 172
    except LookupError as e:
                        ^
SyntaxError: invalid syntax
